pkgs/development/python-modules/moto/default.nix

```
@@ -95,12 +96,20 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     freezegun
     pytestCheckHook
+    pytest-order
     pytest-xdist
   ] ++ passthru.optional-dependencies.all;
 
+  # Some tests depend on AWS credentials environment variables to be set.
+  AWS_ACCESS_KEY_ID = "ak";
+  AWS_SECRET_ACCESS_KEY = "sk";
```

```suggestion
  # Some tests depend on AWS credentials environment variables to be set.
  env.AWS_ACCESS_KEY_ID = "ak";
  env.AWS_SECRET_ACCESS_KEY = "sk";
```

We prefer explicitly leaking things into the environment for a while now.